src/minisatip.c

```diff
@@ -1223,13 +1223,13 @@ int read_rtsp(sockets *s)
 							 //							opts.start_rtp, opts.start_rtp + 1,
 							 get_sock_sport(sid->rsock),
 							 get_sock_sport(sid->rtcp), get_session_id(s->sid),
-							 s_timeout, sid->sid + 1);
+							 s_timeout, sid->sid == 0 ? sid->sid + 1 : sid->sid);
```

Comment:
I feel this `sid->sid == 0 ? sid->sid + 1 : sid->sid` is equal to:
`sid->sid == 0 ? 1 : sid->sid`, right?